### ISSUE TITLE
Cell style configuration is reseted each time I save file

### ISSUE THREAD

**Issue by ihorbach** [EPPlusSoftware/EPPlus#772]
I change some cell style configuration, for example by unchecking checkboxes (they are checked by default) in style properties dialog
(sorry for german screenshot, I don't have any english installation)
![image](https://user-images.githubusercontent.com/33231799/209831805-6256867d-aea8-47fa-9dd9-53a40b7c0587.png)

When I save the file out of excel and open it once again - all changes persist.
If I open and save this file with EPPlus - the changes are gone and all the checkboxes are checked again

**Comment by ihorbach** [comment]
I made a merge request for this issue. Can you please review it?
https://github.com/EPPlusSoftware/EPPlus/pull/773

There was something strange, which I did not understand in your code. I see no reason, why it should be so. But I don't have such a deep knowladge about excel. In any case it works in EPPlus otherway around, as excel does.
Class ExcelXfsXml, function CreateXmlNode. At the very end you have following code snippet:

```
if ((Locked == false || Hidden == false))
{
    SetXmlNodeString("@applyProtection", "1");
}
```

Can you please review it for correctness?

**Comment by JanKallman** [comment]
I have added a fix for this in 6.1.2. I could not use the PR as it would cause issues to LibreOffice that uses the Apply* properties a little bit differently than Excel. Thanks.
